pkg/apis/template/extension.go

```diff
@@ -63,11 +63,17 @@ func (h Handler) RouteGetVersions(
 		query.Select(fields...)
 	}
 
+	//nolint:lll
+	// From https://semver.org/#is-there-a-suggested-regular-expression-regex-to-check-a-semver-string.
+	semverExpression := `^(0|[1-9]\d*)\.(0|[1-9]\d*)\.(0|[1-9]\d*)(?:-((?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\.(?:0|[1-9]\d*|\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?$`
```

Comment:
Do you think we can simply filter out sortable versions by the presence of the `.` charactor?

Reply:
Updated to simplied expression. The short one may parse some invalid semver but is acceptable in our case. From semver offical site it can cover all cases if we want strictly filter semver tags.